### PR TITLE
add styled google map; fix indentation

### PR DESCRIPTION
**Pull request by neu5**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,146 +1,150 @@
...
+          <img src="/img/map-line.png" alt="to jest oddzielająca linia">
+        </li>
+        <li>
+          <ul>
+            <li class="bargar-wrapper">
+              <img src="/img/garage_map_icon.png" alt="garage">
             </li>
-            <li>
-                <ul>
-                  <li class="bargar-wrapper">
```
Comment (Cherreny) [inline index.html:116]: why did you delete garage icon?

Comment (neu5) [inline index.html:116]: I didn't :smiley:

**Comment by neu5** [comment]
Markers are missing :D
